doc/source/rllib-training.rst

```diff
@@ -1,7 +1,8 @@
+.. admonition:: We're hiring!
```

Comment:
huh, rst doesn't support something like include another template?

Reply:
Yeah, was thinking about this, too :D 
Not a sphinx expert, but I think you can define shared blocks and then use those instead. Will fix.

Reply:
fixed